### ISSUE TITLE
[QUESTION] Is it still working with heroku ?

### ISSUE THREAD

**Issue by imTHAI** [benbusby/whoogle-search#701]
Hello,

I used whoogle during several months on Heroku without any problem.
For a personal reason I had to stop it and use this heroku account for something else.
Now I created a new account, I clicked on the link "Deploy on Heroku", it proceeds to the install, then it says it's finished and I can click on Open App, but the page is always an Application Error page with the message "There's nothing here, yet."
I dinde change any variable(yet). Is there something special to do ?

**Comment by benbusby** [comment]
I'll look into it. Are there any error messages in your heroku logs that might indicate what the problem is?

**Comment by imTHAI** [comment]
Hello @benbusby ,

Thank you very much for your time and your work ! I really enjoy whoogle which became my main search engine on all my devices (when possible).
The next day, the page showed up ! I don't know why it did take so much time ( I don't say it toke 24h, but I tried during 15min before I decided to open this issue). Is it possible heroku does something to thwart the wave of deployment of whoogle apps on their servers ? :)
Anyway now it works.

Have a nice day,
